test/configuration/configuration.test.ts

```diff
@@ -1,7 +1,8 @@
 import * as assert from 'assert';
 import * as srcConfiguration from '../../src/configuration/configuration';
 import * as testConfiguration from '../testConfiguration';
-import { cleanUpWorkspace, setupWorkspace } from './../testUtils';
+import * as vscode from 'vscode';
+import { cleanUpWorkspace, setupWorkspace, createRandomFile } from './../testUtils';
```

Comment:
```suggestion
import { cleanUpWorkspace, setupWorkspace } from './../testUtils';
```

I was using it at one point, but `createRandomFile` is no longer needed for the test.